### ISSUE TITLE
Cannot find icon

### ISSUE THREAD

**Issue by MisterMakerNL** [FreeCAD/FreeCAD-render#439]
I get the following warning when starting Render wb:
![image](https://github.com/user-attachments/assets/ffefaf43-b568-4d07-945b-2576ea5f3c06)

**Comment by howetuft** [comment]
Yes, I lazily chose to use FreeCAD icon for the camera, but they changed it recently and the link is now dead. Should be easy to fix, I'll notify you soon.

**Comment by howetuft** [comment]
OK, this should be fixed (please update workbench to test...)

**Comment by howetuft** [comment]
Thanks!
